src/index.html

```diff
@@ -4,9 +4,271 @@
+            <a href="#" class="link">Смартфоны и телефоны</a>
+          </li>
+          <li class ="nav_point" data-qa="nav-hover">
+            <a href="#" class="link">Ноутбуки и компьютеры</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Гаджеты</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Планшеты</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Фото</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container" data-qa="card">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            class="container__product"
+            alt="comp">
+        </a>
+        <h2 class="container__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="container__number">Код товара: 195434</div>
+        <div class="container__stars">
+          <div class="container__stars container_4">
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+          </div>
+          <div class="container__feedback">Отзывов: 5</div>
+        </div>
+        <div class="container__price">
+          <div class="container__price-name">Цена:</div>
+          <div class="container__price-number">69 999 грн</div>
+        </div>
+        <a href="#"
+        class="buy"
+        data-qa="hover"
+        >
+        КУПИТЬ
+        </a>
+      </div>
+      <div class="container">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            class="container__product"
+            alt="comp">
+        </a>
+        <h2 class="container__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="container__number">Код товара: 195434</div>
+        <div class="container__stars">
+          <div class="container__stars container_4">
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+            <div class="container__star"></div>
+          </div>
+          <div class="container__feedback">Отзывов: 5</div>
+        </div>
+        <div class="container__price">
+          <div class="container__price-name">Цена:</div>
+          <div class="container__price-number">69 999 грн</div>
+        </div>
+        <a href="#"
+        class="buy"
+        >
+        КУПИТЬ
+        </a>
+      </div>
+      <div class="container">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            class="container__product"
+            alt="comp">
```

Comment:
```suggestion
          <img 
            src="./images/imac.jpeg"
            class="container__product"
            alt="Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)"
          >
```
Alt желательно делать более подробным, так как по нему пользователи со скринридером понимают что изображено на картинке